app/src/main/java/com/google/android/samples/socialite/SocialApp.kt

```diff
@@ -16,7 +16,17 @@
 package com.google.android.samples.socialite
 
 import android.app.Application
+import com.google.android.samples.socialite.widget.SocialiteWidget
+import com.google.android.samples.socialite.widget.model.WidgetModelRepository
 import dagger.hilt.android.HiltAndroidApp
+import javax.inject.Inject
 
 @HiltAndroidApp
-class SocialApp : Application()
+class SocialApp : Application() {
+
+    @Inject lateinit var repository: WidgetModelRepository
```

Comment:
If you create a custom Hilt `@EntryPoint`, the appplication class wouldn't need to hole a reference to widget dependencies

Reply:
Fixed